openapi_python_client/openapi_parser/openapi.py

```diff
@@ -236,14 +236,16 @@ def _iterate_properties() -> Generator[Property, None, None]:
 
     @staticmethod
     def from_dict(d: Dict[str, Dict[str, Any]], /) -> OpenAPI:
-        """ Create an OpenAPI from dict """
+        """ Create an OpenAPI from dict 
+        :rtype: object
+        """
         schemas = Schema.dict(d["components"]["schemas"])
         endpoint_collections_by_tag = EndpointCollection.from_dict(d["paths"])
         enums = OpenAPI._check_enums(schemas.values(), endpoint_collections_by_tag.values())
 
         return OpenAPI(
             title=d["info"]["title"],
-            description=d["info"]["description"],
+            description=d["info"].get("description"),
```

Comment:
This is to allow description to be empty right?  So then we'd need to make `OpenAPI.description` an `Optiona[str]` instead of a `str` to be correct.